tests/Opus/JobTest.php

```diff
@@ -54,10 +54,11 @@ public function testCreatedJobWritesSha1ToHashColumn()
         $job->setData('somedata');
         $jobId = $job->store();
 
-        $jobTable = TableGateway::getInstance(Jobs::class);
-        $jobRow   = $jobTable->fetchRow("id = $jobId");
+        $sha1Id = $job->generateSha1Id();
```

Comment:
Ist das immer noch der gleiche Test? Ich würde sagen fast. Beim Speichern wird der SHA1 Wert generiert. Er sollte vom ersten Job-Objekt mit `getSha1Id` abgefragt werden können. Er muss dann mit dem Wert in der Datenbank verglichen werden. Dafür wird der Job noch mal aus der Datenbank ausgelesen. Ob das über den EntityManager Sinn macht, kann ich nicht beurteilen. Ich denke aber hier sollte `getSha1Id` verwendet werden. Die `generateSha1Id` könnte im Prinzip auch **protected** sein.